_posts/blog/2014-12-30-stress-test-long-run.md

```diff
@@ -22,19 +22,19 @@ everything finished, the test was a success. The number of unique integers
 computed independently by MapReduce matched the number computed by Fluo. Below
 is the output of the stress test count from Fluo.
 
-     $ java -cp $STRESS_JAR io.fluo.stress.trie.Print $FLUO_PROPS
+     $ java -cp $STRESS_JAR stresso.trie.Print $FLUO_PROPS
```

Comment:
I agree with Keith. Blog posts should be updated only in special situations (i.e to fix a broken link or correct wrong information).  I don't think blog posts should be updated due to code changes in later Fluo releases.